Tests/WorkspaceTests/SourceControlPackageContainerTests.swift

```diff
@@ -510,6 +510,18 @@ class SourceControlPackageContainerTests: XCTestCase {
             )
         }
 
+        let v5_2ProductMappingWithoutCommands: [String: ProductFilter] = [
```

Comment:
The differing context means the `includeCommands` need to be opposite what they were above, so they can no longer share the same variable.